sci-mathematics/octave/files/octave-6.4.0-omit-qtchooser-qtver.patch

```diff
@@ -0,0 +1,63 @@
+diff -urNp octave-6.4.0-orig/configure octave-6.4.0-dwok/configure
```

Comment:
This is still only patching configure, not configure.ac, which is fragile -- and we're already calling eautoreconf anyway.

Also, given this patch is applied before eautoreconf (which is right), the changes are likely to get blown away, so this does not do anything right now, I think.

Just change it to only patch configure.ac then test it.